pkgs/by-name/li/libmsquic/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+lib
+, stdenv
+, fetchFromGitHub
+, cmake
+  , lttng-tools
+  , libatomic_ops
+  , perl
+  , coreutils
+, ...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libmsquic";
+  version = "2.4.7";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "msquic";
+    rev = "v${version}";
+    sha256 = "WveyZ9rMevLTc5C4cgMFcnj0O6Hd+HcfU8ccD6VBgyU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake perl coreutils ];
+
+  buildInputs = [ lttng-tools libatomic_ops ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
This is handled automatically, not needed.

Reply:
Done